docs/source/models/adding_model.rst

```diff
@@ -94,3 +96,16 @@ This method should load the weights from the HuggingFace's checkpoint file and a
 ----------------------
 
 Finally, include your :code:`*ForCausalLM` class in `vllm/model_executor/models/__init__.py <https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/__init__.py>`_ and register it to the :code:`_MODEL_REGISTRY` in `vllm/model_executor/model_loader.py <https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/model_loader.py>`_.
+
+6. Out-of-Tree Model Integration
+--------------------------------------------
+
+We also provide a way to integrate a model without modifying the vLLM codebase. Step 2, 3, 4 are still required, but you can skip step 1 and 5.
+
+Just add the following lines in your code:
+
+.. code-block:: python
+
+    from vllm.model_executor.models import ModelRegistry
```

Comment:
Why don't we expose `ModelRegistry` so that users can do
```python
from vllm import ModelRegistry
```

Reply:
fixed in [9b6655e](https://github.com/vllm-project/vllm/pull/3871/commits/9b6655e0a8c1f22a6c67c7e2bd6973d9971b6481)